HTML/QuickForm/hierselect.php

```diff
@@ -120,7 +120,7 @@ class HTML_QuickForm_hierselect extends HTML_QuickForm_group
      */
     function __construct($elementName=null, $elementLabel=null, $attributes=null, $separator=null)
     {
-        parent::__construct($elementName, $elementLabel, $attributes);
+        parent::__construct($elementName, $elementLabel, null, null, null, $attributes);

```

Comment:
Spot-checked: I grepped for examples of `hierselect`. Picked one from "Administer => Customize => ... => Add Custom Field" (data type)  and still works. It still works in PHP 5.3.